README.md

```diff
@@ -1 +1,200 @@
-# react-capstone
\ No newline at end of file
+# reactcapstone
+<a name="readme-top"></a>
+
+<div align="center">
+
+  # THIERRY NOUMBUH
+  <br/>
+
+  <h3><b>REACT-METRICS</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo / Video ShowCase](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 react-capstone
+
+This React capstone project is about building a mobile web application to check a list of metrics (numeric values) that I created making use of React and Redux.
+
+![Preview Image](Images/Preview.PNG)
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Frontend developer</summary>
+  <ul>
+    <li>REACT js</li>
+    <li>CSS</li>
+    <li>Node js</li>
+    <li><a href="https://fonts.googleapis.com"><li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- Added Slice file
+- Added tailwind.css file
+- Added index.js file
+- Included appropriate Linters.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo](Coming soon)
+- [Live Demo](https://axcell-0.github.io/react-capstone//)
+- [Video ShowCase](https://drive.google.com/file/d/1Th79Z6mTNnOaYIT7hF_wf7MIxP3D58Oz/view?usp=sharing)
+
+<p align="right">(<a href="#">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. A Code Editor like:
+- VSCode
+- Atom, etc.
+
+2. NodeJs extension installed in your code editor.
+
+3. A Web Browser like
+- Google Chrome
+- Firefox
+- Microsoft Edge
+- Opera, etc.
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+git clone https://github.com/Wils-FOURR-speed/react-metrics-capstone.git
+
```

Comment:
- [ ]  Can you make available the link to the project to be cloned? Because the link you provide does not belong to this project. Kindly make the change to avoid confusion